agent/common.sh

```diff
@@ -1,8 +1,7 @@
 #!/usr/bin/env bash
 set -euxo pipefail
 
-export FLEETING_PATH=${FLEETING_PATH:-$WORKING_DIR/fleeting}
-export FLEETING_ISO=${FLEETING_ISO:-$FLEETING_PATH/output/fleeting.iso}
-export FLEETING_MANIFESTS_PATH="${FLEETING_PATH}/manifests"
-export FLEETING_PR=${FLEETING_PR:-}
+export INSTALLER_PATH="${OPENSHIFT_INSTALL_PATH:-${HOME}/go/src/github.com/openshift/installer}"
+export FLEETING_ISO=${FLEETING_ISO:-$INSTALLER_PATH/output/fleeting.iso}
+export FLEETING_MANIFESTS_PATH="${INSTALLER_PATH}/manifests"
```

Comment:
Even though initially we'll be forced to build always the installer from the src, I think it would be preferrable to copy the manifests/artifacts in the `WORKING_DIR` and continue to use the one found there. Probably we could start creating an `agent` folder within the `WORKING_DIR` to store such artifacts

Reply:
Another point: maybe it could be useful (even though with some additional maintenance effort) to maintain the previous fleeting support for a while, until the installer code will be consolidated, instead of throwing it away immediately?